stock_inventory_line_open/models/stock.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockInventory(models.Model):
+    _name = "stock.inventory"
+    _inherit = ["stock.inventory", "mail.thread"]
+
+    state = fields.Selection(tracking=True)
+
+    def _track_subtype(self, init_values):
```

Comment:
Maybe you could add something in README for that no ?

Reply:
ups, that is dead code (not imported), I will remove it.